_posts/2016-03-31-autoconfiguration.md

```diff
@@ -3,9 +3,9 @@ layout: posts
 title: Using Skylark remote repositories to auto-detect the C++ toolchain.
 ---
 
-[Skylark remote repositories](/docs/skylark/repository_rules.html) let you
-create custom [external repositories](/docs/external.html) using
-[Skylark](/docs/skylark/index.html). This not only enables creating rules for
+[Skylark remote repositories](site.docs_site_url/skylark/lib/skylark/repository_rules.html) let you
```

Comment:
Variables need to be surrounded with braces: `{{ site.docs_site_url }}`

Reply:
appears done